org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/services/XMLCompletions.java

```diff
@@ -288,14 +293,53 @@ private CompletionList collectAttributeValueSuggestions(int valueStart, int valu
 		return completionResponse;
 	}
 
-	private CompletionList collectOpenTagSuggestions(int tokenOffset, int tokenEnd) {
-		// TODO Auto-generated method stub
-		return null;
+	//Tags that look like '<{CHARS}'
+	private CompletionList collectOpenTagSuggestions(int tokenOffset, int tokenEnd, String tag) {
+		XMLDocument xmlDocument = completionRequest.getXMLDocument();
+		List<CompletionItem> list= new ArrayList<CompletionItem>();

```

Comment:
declare that list right before using it (L310), else it might be useless resource allocation